exportTestEnvironments.js

```diff
@@ -0,0 +1,43 @@
+import dotenv from 'dotenv';
+import * as childProcess from "child_process";
+import minimist from 'minimist';
+
+const { i, s, t } = minimist(process.argv.slice(2));
+
+dotenv.config();
+
+const { API_KEY, SYNC_SOURCE_TEST_ENVIRONMENT_ID, SYNC_TARGET_TEST_ENVIRONMENT_ID, EXPORT_IMPORT_TEST_DATA_ENVIRONMENT_ID } = process.env;
+
+if (!i && !s && !t) {
+  console.log("You need to atleast one of the arguments: -i for Import Environment, -s for Sync Source Environment, -t for Sync Target Environment");
```

Comment:
ditto in earlier readme comment. I would adjust this to set all flags to true if none are specified. if the desired behavior is indeed to log an error message, then adjust the wording and fix typos:

> You need to **provide at least**...